flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/sink/filesystem/Buckets.java

```diff
@@ -153,6 +168,10 @@ private void initializeActiveBuckets(final ListState<byte[]> bucketStates) throw
 	private void handleRestoredBucketState(final BucketState<BucketID> recoveredState) throws Exception {
 		final BucketID bucketId = recoveredState.getBucketId();
 
+		if (LOG.isDebugEnabled()) {
+			LOG.debug("Restoring: {}", recoveredState);
```

Comment:
I would generally try to have each logging statement by self-contained, i.e. this one should also contain the subtask index for example.